### ISSUE TITLE
Dies on scrambled .puz files

### ISSUE THREAD

**Issue by balshetzer** [downforacross/puzjs#7]
This prevents scrambled .puz files from loading on downforacross.com

![Screen Shot 2020-04-28 at 2 28 40 PM](https://user-images.githubusercontent.com/1846726/80523772-994a4780-895c-11ea-9f1b-a5e045573895.png)

Here is a sample puz file that won't load:
https://drive.google.com/open?id=1qEcIktI_sRsmxzRP-vz20N6o9wTv-XdT
from
https://pgwcc.net/2020/04/28/puzzle-57-solution-and-puzzle-58-double-header/

**Comment by stevenhao** [comment]
Next steps: Check if other applications successfully read this file
A few
- https://github.com/alexdej/puzpy
- https://www.litsoft.com/across/alite/download/

**Comment by balshetzer** [comment]
Yes. It opens fine in other apps. In fact I used puzpy to remove the scrambling and then opened it in downforacross. 

This library throws an error if the solution is scrambled. The library should either implement descrambling like pypuz or (much easier) just treat it like a .puz without solution data.
